repo/fsrepo/fsrepo.go

```diff
@@ -164,30 +180,17 @@ func Init(repoPath string, conf *config.Config) error {
 
 // Remove recursively removes the FSRepo at |path|.
 func Remove(repoPath string) error {
-	repoPath = path.Clean(repoPath)
```

Comment:
`path.Clean` should never change the filesystem semantics of a path, but I'll put the cleaning back in just to make it read the same way.